package-lock.json

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
I think this lock file is misplaced?

Reply:
where it should be placed?

Reply:
in seems to have atleast the fonts, so inside the cc-ui. maybe undo changes of this file, and run the font installations inside the cc-ui folder?

Reply:
or actually, it seems that the upper level does not have it's own package-json, so this file should be deleted and the dependencies (at least fonts?) should be installed inside the cc-ui folder